ttnn/ttnn/operations/binary.py

```diff
@@ -486,6 +486,7 @@ def add_and_apply_activation(
     activation: Optional[str] = None,
     memory_config: ttnn.MemoryConfig = ttnn.DRAM_MEMORY_CONFIG,
     dtype: Optional[ttnn.DataType] = None,
+    in_place: Optional[bool] = False,
```

Comment:
We already have an in-place version. With the underscore at the back.use that and don't modify this